File: Projects/server/Contracts/Requests/CreateTelemetryDataRequest.cs
namespace server.Contracts.Requests
{
    public class CreateTelemetryDataRequest
    {
        public DateTime RecDatetime { get; set; }
        public byte ServiceId { get; set; }
        public ushort PID { get; set; }
        public byte[] EcuId { get; set; } = null!;
        public byte ResponseDlc { get; set; }
        public byte[]? Response { get; set; }
        public ulong TripId { get; set; }
    }
}

File: Projects/DataProviderApp/app/src/main/java/com/example/dataproviderapp/dto/requests/CreateTelemetryDataRequest.kt
package com.example.dataproviderapp.dto.requests

data class CreateTelemetryDataRequest(
    val recDatetime: String,
    val serviceId: UByte,
    val pid: UShort,
    val ecuId: String,
    val responseDlc: UByte,
    val response: String?,
    val tripId: ULong
)
